src/templates/en/2019/base_chapter.html

```diff
@@ -32,6 +32,14 @@
   var el = document.getElementById('num_comments');
   el.innerText = comments + ' ' + (comments == 1 ? 'comment' : 'comments');
 });
+document.addEventListener("keypress", function onPress(event) {
+ 	if (event.key === 'p' || event.key === ',') {
```

Comment:
Should we cater for other versions of these keypresses?

```suggestion
 	if (event.key === 'p' || event.key === 'P' || event.key === ',' || event.key === '<') {
```

Reply:
We could include check for the uppercase but I think we should not include the arrow("<", ">") keys for this as it's a bit counter-intuitive ( cause arrows are also used for vertical and horizontal scrolling).

Reply:
In this case we're only targeting angled brackets, not the actual arrow keys. I think it's ok to handle `Shift` key versions of other supported shortcut keys.